src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
Yeah, looks like I'm out voted on this one. I do like a few additional steps we discussed.
- exporting each function when it's created instead of a giant object at the end of the file
- breaking up lib into smaller focused libs, e.g. dom functions in separate file